curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-an-ordered-list.english.md

```diff
@@ -8,7 +8,7 @@ videoUrl: 'https://scrimba.com/p/pVMPUv/cQ3B8TM'
 ## Description
 <section id='description'>
 HTML has another special element for creating <code>ordered lists</code>, or numbered lists.
-Ordered lists start with an opening <code>&#60;ol&#62;</code> element, followed by any number of <code>&#60;li&#62;</code> elements. Finally, ordered lists close with a <code>&#60;/ol&#62;</code>
+Ordered lists start with an opening <code>&#60;ol&#62;</code> element, followed by any number of <code>&#60;li&#62;</code> elements. Finally, ordered lists close with an <code>&#60;/ol&#62;</code>.
```

Comment:
I recommend the following changes:
```suggestion
Ordered lists start with an opening <code>&#60;ol&#62;</code> element, followed by any number of <code>&#60;li&#62;</code> elements. Finally, ordered lists are closed with the <code>&#60;/ol&#62;</code> tag.
```

Reply:
@VedranaR I like @cmccormack's suggestion, because it eliminates the debate over `a` vs. `an`.  Please make this change, so we can merge this PR.

Reply:
@cmccormack I have made your suggested change on behalf of the user.  I am fine with the current state of this PR.  Please merge if you are ok with it, as there are no earlier PRs for this challenge file.